samples/net/zperf/src/zperf_shell.c

```diff
@@ -658,8 +662,12 @@ static int execute_upload(const struct shell *shell,
 	}
 
 out:
+if (IS_ENABLED(CONFIG_NET_IPV6)) {
```

Comment:
Here, you use `if (IS_ENABLED` as if it was `#ifdef`. But it's normal C statement, and thus should be indented.